src/stc/stc.cpp

```diff
@@ -224,9 +224,8 @@ bool wxStyledTextCtrl::Create(wxWindow *parent,
     // STC doesn't support RTL languages at all
     SetLayoutDirection(wxLayout_LeftToRight);
 
-    // Rely on native double buffering by default, except under Mac where it
-    // doesn't work for some reason, see #18085.
-#if wxALWAYS_NATIVE_DOUBLE_BUFFER && !defined(__WXMAC__)
+    // Rely on native double buffering by default.
+#if wxALWAYS_NATIVE_DOUBLE_BUFFER
```

Comment:
This looks like unintentional change.

Reply:
Yes, this is really strange. I did a fresh git clone before applying my modified patch (which doesn't touch stc.cpp) and somehow this is in the diff.

The only way I could clear it was by a git reset.

Anyway, it should definitely not be here.